src/shared/errors/unprocessable-entity.error.ts

```diff
@@ -0,0 +1,10 @@
+export class UnprocessableEntity extends Error {
+  statusCode = 402;
```

Comment:
If you refer to HTTP code, it is `422`, not `402`. Furthermore, if I read the message it seems that the problem is that the entity doesn't exist, instead the name of the class refers to another problem: use different errors.